### ISSUE TITLE
producing bad output

### ISSUE THREAD

**Issue by eric-g-97477** [mbloch/mapshaper#579]
I am using `Admin 0 – Details > map units` [10m NaturalEarthData](https://www.naturalearthdata.com/downloads/10m-cultural-vectors/) ( [file](https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_map_units.zip) ) 

I figure there is a multitude of tools out there that would do the job. 

I ran a couple of the commands mentioned in the documentation and geopandas was unable to load the simplified data successfully. It can load the original data without issue.

The commands were:

```
mapshaper ne_10m_admin_0_map_units.geo.json -simplify 10% -o simplified.geo.json
mapshaper ne_10m_admin_0_map_units.geo.json -simplify interval=100 -o simplified.geo.json
```

The geopandas code was:

```
gdf = geopandas.GeoDataFrame.from_features( simplifiedData, crs = 4326 )
gdf.explore()
```

and geopandas generated the error:

```
folium/features.py:1099, in <listcomp>(.0)
   1094 def warn_for_geometry_collections(self):
   1095     """Checks for GeoJson GeometryCollection features to warn user about incompatibility."""
   1096     geom_collections = [
   1097         feature.get("properties") if feature.get("properties") is not None else key
   1098         for key, feature in enumerate(self._parent.data["features"])
-> 1099         if feature["geometry"]["type"] == "GeometryCollection"
   1100     ]
   1101     if any(geom_collections):
   1102         warnings.warn(
   1103             "{} is not configured to render for GeoJson GeometryCollection geometries. "
   1104             "Please consider reworking these features: {} to MultiPolygon for full functionality.\n"
   (...)
   1108             UserWarning,
   1109         )

TypeError: 'NoneType' object is not subscriptable
```

Is this a bug in mapshaper? Is there a correct way I should be using mapshaper? Is there a problem with the NaturalEarthData?

**Comment by eric-g-97477** [comment]
Playing around with it more, I believe the reason is that during the simplification process, some of the features were simplified out of existence. When I added keep-shapes to the command, the output mapshaper produced was correct. 

I do believe there is a bug here because mapshaper should not be outputting bad data. 

The shape file I pointed to is trivial to convert to json using geopandas.

```
    shapefile = geopandas.read_file( shapeFilePath )
    shapefile.to_file( outPath, driver="GeoJSON" )  
```
